src/services/indexer/subsquid/client.ts

```diff
@@ -0,0 +1,49 @@
+import { createClient, defaultExchanges, subscriptionExchange } from '@urql/core';
+import { createClient as createWSClient } from 'graphql-ws';
+
+import type { Client } from '@urql/core';
+
+export type { Client, OperationResult, TypedDocumentNode, AnyVariables } from '@urql/core';
+
+const createSubscriptionClient = (url: string) => {
+  const wsUrl = url.replace(/^http/, 'ws');
+
+  return createWSClient({
+    url: wsUrl,
+    lazy: true,
+    retryAttempts: 25,
```

Comment:
do we have the same configuration for subsquid? maybe it's better to move it somewhere in common place? what do you think?

Reply:
Moved to consts/indexer.ts